src/server/dragonfly_test.cc

```diff
@@ -438,6 +440,37 @@ TEST_F(DflyEngineTest, PSubscribe) {
   EXPECT_EQ("ab", msg.channel);
   EXPECT_EQ("a*", msg.pattern);
 }
+TEST_F(DflyEngineTest, Unsubscribe) {
+  auto resp = pp_->at(1)->Await([&] { return Run({"unsubscribe", "a"}); });
```

Comment:
since you use a single connection in this test, you may simplify the code here.
instead  of `auto resp = pp_->at(1)->Await([&] { return Run({"unsubscribe", "a"}); });` you can write `auto resp = Run({"unsubscribe", "a"});`. Should work if I do not miss anything.
It should be consistent with other calls in this test.

Reply:
Thanks! That does indeed make the tests much more compact!